pkg/controller/virtualmachineinstance/controller.go

```diff
@@ -75,6 +76,15 @@ func (h *Handler) OnVmiChanged(_ string, vmi *kubevirtv1.VirtualMachineInstance)
 		return vmi, nil
 	}
 
+	// Skip processing if the external-managed-node label is present
```

Comment:
@mattmattox  Thanks for your committing and sorry for the later review, as no one was assigned to review the PR.

The PR looks good, and I only have this question:

L88 has the check `if creator := vmi.Labels[builder.LabelKeyVirtualMachineCreator]; creator != virtualmachine.VirtualMachineCreatorNodeDriver` to skip the VM.

If you add external VMIs to the cluster, does the related VMI still happens to have `LabelKeyVirtualMachineCreator` label? could you utilize this existing label to skip it?